### PR TITLE
Update SPO-asynchronous-read-API.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue #995 - Missing period and grammar issues are corrected.

### CODE REVIEW

**Comment by scanum** [comment]
@JoanneHendrickson Changes look correct. Could you please review and merge? Thanks!

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/SPO-asynchronous-read-API.md [hunk 1]
```diff
@@ -45,11 +45,7 @@ The API is made up of two input parameters url,  and an Optional Flag  and one o
 #### URL 
 The URL allows SPMT or third party tools to specify the root URL path of the Sharepoint objects to be read.  The server side code will read and return all the subfolder/files/list of that root URL.
 
-For example, if for a document library url: https://www.contoso.com/my-resource-document/
-for any files or folder shares the same root URL, the supporting content will be read back 
-https://www.contoso.com/my-resource-document/file1.doc or 
-https://www.contoso.com/my-resource-document/folderA/file2.doc 
-Only the root url needs to be specified and sends as a single read request.
+For example, if a document library URL is https://www.contoso.com/my-resource-document/, any files or folders share the same root URL, their individual URLs will look like https://www.contoso.com/my-resource-document/file1.doc or https://www.contoso.com/my-resource-document/folderA/file2.doc. Only the root url needs to be specified and it should be sent as a single request.
```
Comment (JohanFreelancer9) [inline migration/SPO-asynchronous-read-API.md]:
```suggestion
For example, if a document library URL is https://www.contoso.com/my-resource-document/, any files or folders share the same root URL, and their individual URLs will look like this: https://www.contoso.com/my-resource-document/file1.doc or https://www.contoso.com/my-resource-document/folderA/file2.doc. Only the root URL needs to be specified and it should be sent as a single request.
```

**Comment by scanum** [comment]
@JoanneHendrickson Hi Joanne, any updates, PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@JoanneHendrickson Hi Joanne, any updates, PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Pull request merged**